modules/core/src/main/scala/org/scalasteward/core/update/FilterAlg.scala

```diff
@@ -133,6 +133,11 @@ object FilterAlg {
           // https://github.com/lightbend/migration-manager/pull/260
           "3.2.0-SNAP10"
         )
+      case ("io.circe", _, _, _) =>
```

Comment:
This will mark all circe 0.12.0 modules as bad. Even those that don't live in the `circe/circe` repo.

Reply:
Hmm, yeah, I guess that's not ideal. I'll scope it down.